There's one last piece of feedback from the linter:
```
kolibri/plugins/utils/__init__.py:487:12: E721 do not compare types, for exact checks use `is` / `is not`, for instance checks use `isinstance()`
```

--- a/kolibri/plugins/utils/__init__.py
+++ b/kolibri/plugins/utils/__init__.py
@@ -484,7 +484,7 @@ def iterate_plugins():
     all_entry_points = entry_points()
     eps = (
         all_entry_points.get("kolibri.plugins", [])
-        if type(all_entry_points) == dict
+        if isinstance(all_entry_points, dict)
         else all_entry_points.select(group="kolibri.plugins")
     )
     for entry_point in eps:

